aws/aws-explanation/ec2-image-retention-policy.rst

```diff
@@ -3,7 +3,7 @@ AWS image retention policies
 
 Ubuntu images on AWS have three stage of life cycle : *release* > *deprecation* > *deletion*. 
 
-Whenever a new image is built and released to AWS, all the previous serials for that image are deprecated except for the last 3. A deprecated image is not visible on the AWS console, but it can still be launched from the `AWS CLI`_ using its AMI ID. You can also find and view details of deprecated images using the AWS CLI `describe-images`_ command by including the ``--include-deprecated`` flag.
+Whenever a new image is built and released to AWS, all the previous serials for that image are deprecated except for the last 3. A deprecated image is not visible on the AWS console, but it can still be launched from the `AWS CLI`_ using its AMI ID. You can also find and view details of deprecated images using the AWS CLI `describe-images`_ command by including the ``--include-deprecated`` flag. Unlike deprecated images, private images are not available to use and invisible to describe-images. This step of marking an image private is reserved for AMIs that go unused and have been long since replaced by newer images.
```

Comment:
Can we add a bit more explanation about private images - what are they used for? Maybe something like:
 
```suggestion
Whenever a new image is built and released to AWS, all the previous serials for that image are deprecated except for the last 3. A deprecated image is not visible on the AWS console, but it can still be launched from the `AWS CLI`_ using its AMI ID. You can also find and view details of deprecated images using the AWS CLI `describe-images`_ command by including the ``--include-deprecated`` flag.

Apart from these three stages, some AMIs also become *private*. AMIs that go unused and have long since been replaced by newer images are marked as private. This is done so that ......  Unlike deprecated images, private images are not available for use and are invisible to the ``describe-images`` command. 
```
Also add something about when they get deprecated / deleted. That part is not clear from the table.